### PR TITLE
Laze loadoutstuff

### LINKED ISSUE #638: Tanoa/Apex vanilla templates currently not being called
*Version:* 2.2

***Issue:***
None of the three Tanoa/Apex templates (with the proper occupant militia units, for example) are currently being called in 2.2. Something was probably lost in translation.

We could do with a better method of selecting modsets, but a temporary fudge will do for 2.2.

### PR DESCRIPTION
**Pull request by Lazejun**
## What type of PR is this.
1. [x] Bug
2. [x] Enhancement

### What have you changed and why?
Information:
-Added Call for Tanoa in Occ/Inv Altis Templates.
-Added _Altis/_Tanoa Loadouts for BlueforVanilla.
-Removed Contact DLC Radiobackpack.
-Altis Loadouts are DLC/Watermarkfree.
-Gave AT Rifleman Maaws Mk4 Mod 0 with 2 HEAT55 Rockets.
-Altis Rifleman get Black MXes.
-Altis Autorifleman gets Mk200.
-Altis Marksman gets Mk14 with SOS Scope (2,5-5X).
-Changed fn_getLoadout.sqf to give 2 Ace Flashbangs and Removed 1 Frag grenade it would have given

### Please specify which Issue this PR Resolves.
closes #638 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the Mission in Singleplayer?
2. [X] Have you loaded the Mission as Local Host?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

Maybe Adjustments on Loadouts but they should be allright, tried to keep them Similar to the previous Vanilla ones.

********************************************************
Notes:

### CODE REVIEW

**Comment by Spoffy** [comment]
Requesting Meerkat's review (because templates), before I go over and check the logic.

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_AT_tanoa.sqf [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_TNA_F", "V_PlateCarrier2_TNA_F"],
+		[//Inventory
+			["NVGoggles_tna_F",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
+			["ACE_M84",2,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_AT_tanoa.sqf]: Same as last

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_rgr", "V_PlateCarrier2_rgr"],
+		[//Inventory
+			["NVGoggles",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf]: This time a vanilla one, does the ace m84 below automatically switch for a vanilla? @Spoffy

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_tanoa.sqf [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_TNA_F", "V_PlateCarrier2_TNA_F"],
+		[//Inventory
+			["NVGoggles_tna_F",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_tanoa.sqf]: nade

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_medic_altis.sqf [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_rgr", "V_PlateCarrier2_rgr"],
+		[//Inventory
+			["NVGoggles",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_medic_altis.sqf]: nade

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_medic_tanoa.sqf [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_TNA_F", "V_PlateCarrier2_TNA_F"],
+		[//Inventory
+			["NVGoggles_tna_F",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_medic_tanoa.sqf]: nade

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_rifleman_altis.sqf [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_rgr", "V_PlateCarrier2_rgr"],
+		[//Inventory
+			["NVGoggles",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_rifleman_altis.sqf]: nade

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_rifleman_tanoa.sqf [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_TNA_F", "V_PlateCarrier2_TNA_F"],
+		[//Inventory
+			["NVGoggles_tna_F",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_rifleman_tanoa.sqf]: nade

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_teamLeader_altis.sqf [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_rgr", "V_PlateCarrier2_rgr", "V_PlateCarrierSpec_mtp"],
+		[//Inventory
+			["NVGoggles",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_teamLeader_altis.sqf]: nade

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_teamLeader_tanoa.sqf [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+		[] + _basicMedicalSupplies + _basicMiscItems
+	],
+
+	[//Vest
+		selectRandom										//Vest
+		["V_PlateCarrier1_TNA_F", "V_PlateCarrier2_TNA_F", "V_PlateCarrierSpec_TNA_F"],
+		[//Inventory
+			["NVGoggles_tna_F",1],
+			["SmokeShell",2,1],
+			["HandGrenade",1,1],
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/vanilla_blufor_teamLeader_tanoa.sqf]: nade

**Comment by Lazejun** [comment]
Removed "Handgrenade" from fn_getLoadout.sqf, which didnt work before and gives 2 Flashbangs now, allready Tested and applied.

File: A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf (hunk 2 above)
Comment (jaj22) [reply A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf]: It does, yeah. Line 86 fn_getLoadout.sqf. So the vanilla HandGrenades probably shouldn't be in here.

Comment (Lazejun) [reply A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf]: That script didnt work before.
Below i have the Change called "Ace_M84 fix ":
Before:
private _aceM84 = [hasACE, ["ACE_M84", 1], ["HandGrenade", 1]] call _fnc_modItem;

private _aceM84 = [hasACE, ["ACE_M84",2,1]] call _fnc_modItem;
this gives 2 Flashbangs and doesnt try to give Handgrenades anymore

Comment (Spoffy) [reply A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf]: So is one of the aims here to remove all handgrenades from PvP loadouts? Wondering based on Meerkat's nade comments

Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/Loadouts/vanilla_blufor_machineGunner_altis.sqf]: Just marking each one so we know where they all are :)

**Pull request merged**